torch/_tensor_docs.py

```diff
@@ -1834,6 +1859,17 @@ def add_docstr_all(method, docstr):
 See :func:`torch.isreal`
 """)
 
+add_docstr_all('is_coalesced',
+               r"""
+is_coalesce() -> bool
+
+Returns True if :attr:`self` tensor is a sparse COO tensor which has
+been coalesced, see :meth:`coalesce`.  If :attr:`self` is not a
```

Comment:
"which has been coalesced, see :meth:\`coalesce\`" -> "that is coalesced. See :meth:\`coalesce\`."

Reply:
fixed.